lib/src/test/java/space/taran/arklib/utils/FileUtilsTest.kt

```diff
@@ -0,0 +1,192 @@
+package space.taran.arklib.utils
+
+import android.util.Log
+import dev.arkbuilders.arklib.utils.*
+import io.mockk.every
+import io.mockk.mockkStatic
+import io.mockk.unmockkAll
+import kotlinx.coroutines.runBlocking
+import org.junit.After
+import org.junit.Assert.*
+import org.junit.Before
+import org.junit.Test
+import java.io.IOException
+import java.nio.file.Files
+import java.nio.file.Path
+import kotlin.io.path.Path
+
+class FileUtilsTest {
+
+    private lateinit var testDir: Path
+    private lateinit var rootFolder: Path
+    private lateinit var childFolder: Path
```

Comment:
Let's use word "favorite" instead of "child". It's project-specific terminology.. "Root" folder is where an index is stored, "favorites" are children of a root folder.

Reply:
Can be "fav" for short.